docs/s3_filesystem.md

```diff
@@ -0,0 +1,47 @@
+---
+title: S3 Filesystem (experimental)
```

Comment:
This is great.. Can we link this to a section under configuration for filesystem config ?